src/functions_framework/__init__.py

```diff
@@ -65,6 +65,21 @@ def __init__(
         self.data = data
 
 
+class _LoggingHandler(object):
+    """Logging replacement for stdout and stderr in GCF Python 3.7."""
+
+    def __init__(self, level, stderr):
+        self.level = level
+        self.stderr = stderr
+
+    def write(self, out):
+        self._log_with_severity(out)
```

Comment:
This should return the number of characters written: https://docs.python.org/3/library/io.html#io.TextIOBase.write

Reply:
Modified write() to return this